### DISCUSSION TITLE
Weekly report apprise

### DISCUSSION THREAD

**Discussion by MueJosh** [mcguirepr89/BirdNET-Pi#1153] [General]
Since the beginning of using BirdNET-Pi and apprise notifications I can't get a weekly report. 
I have toggled it in settings - notifications - weekly report 
I can view the weekly report but I don't receive it at any day/week (with apprise) 

I'm using apprise to send notifications to my telegram channel. 

Does it work in general or is this a problem that only occurs on my birdnetpi? 

Cheers

**Comment by auggy** [comment]
I can see the weekly report via the url but it doesn’t get sent to me.

**Reply by MueJosh** [reply]
Yep, same here..

**Reply by auggy** [reply]
By any chance are you trying to post  to Mastodon

**Reply by MueJosh** [reply]
> By any chance are you trying to post  to Macedon?  

You meant mastodon, right? 

No.. Telegram 
I switched to the fork from @Nachtzuster since it's still getting updates, features and is nicely maintained. 
Maybe it works there (wayyy newer apprise version!) 

I solved my issue, I explained it [HERE](https://github.com/Nachtzuster/BirdNET-Pi/issues/26#issuecomment-2038237141)





if I would know how to make a pull request to the original repo I would do it, right now I made one for the  Nachtzuster birdnet-pi fork
